notificationapi_python_server_sdk/notificationapi.py

```diff
@@ -44,3 +44,9 @@ def send(params):
 
 def retract(params):
     request("POST", "sender/retract", params)
+
+def create_sub_notification_id(params):
```

Comment:
I think these function names would be clearer:

* create_sub_notification
* delete_sub_notification